src/App.js

```diff
@@ -0,0 +1,22 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Header from './features/header';
+import './App.css';
+import Home from './features/home';
+import Details from './features/details';
+
+const App = () => (
+  <>
+    <BrowserRouter>
+      <Header>
+        <Routes>
+          <Route path="/" element={<Home />} />
+          {/* <Route path="/details" element={<Details />} /> */}
```

Comment:
- [ ] please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.